src/cow-react/common/pure/CancelButton/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { LinkStyledButton } from 'theme'
+import { CancellationModal } from './CancellationModal'
+
+// TODO: add for each type when implemented
+export type CancelButtonType = 'soft'
+
+export type CancelButtonContentProps = {
```

Comment:
Shouldn't we drop the `Content` part in the name?  

We have been naming before with the same name pure components and containers. And normally it has the name of the component, in this case `pure/CancelButton`

Reply:
I tried to differentiate to not have two components with the exact same name, and this pattern is used elsewhere, but I'm fine changing it

Reply:
And I forgot to update the file name when I renamed the component